### ISSUE TITLE
[Routing] Routeplanning langs wegen met hogere categorie minder aantrekkelijk maken

### ISSUE THREAD

**Issue by xivk** [anyways-open/functional-routeplanner-ovl#14]
Deze issue komt nog altijd terug in de feedback op de routeplanner:

https://trello.com/c/nR1Cjcsy/73-fietspad-lane-langs-hogere-weg-categorie-minder-aantrekkelijk-maken

**Comment by pietervdvn** [comment]
https://www.fietsrouteplanner.org/#map=11.98/3.67063/50.99749&route=bicycle.commute,Grote%20steenweg%2022%2C%209840%20De%20Pinte/3.68151/50.99310,Zwijnaardsesteenweg%20318%2C%209000%20Gent/3.72216/51.02861

![image](https://user-images.githubusercontent.com/1466478/147365661-8ac452e3-b546-4ca5-812f-b2cae041b1fb.png)

The problem here is that the cyclelanes are drawn out seperately...

**Comment by xivk** [comment]
True, but not this section: https://www.openstreetmap.org/#map=19/51.00739/3.69975
